src/plugins/discover/public/customizations/customization_types/flyout_customization.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export interface FlyoutDefaultActionItem {
+  disabled?: boolean;
+}
+
+export interface FlyoutDefaultActions {
+  viewSingleDocument?: FlyoutDefaultActionItem;
+  viewSurroundingDocument?: FlyoutDefaultActionItem;
+}
+
+export interface FlyoutActionItem {
+  content: React.ReactNode;
+  disabled: boolean;
+}
+
+export interface FlyoutCustomization {
+  id: 'flyout';
+  actions: {
+    defaultActions?: FlyoutDefaultActions;
+    getActionItems?: () => FlyoutActionItem[];
```

Comment:
```suggestion
```
This would make sense, but it doesn't look like it's implemented or used currently, so I think we should drop this for now.

Reply:
You're right, I totally forgot to apply the custom action items into the rendered list 🙈 
Thanks for the heads up, instead of removing it I used it when computing the actions list, as we'll probably need to use it soon for our customizations and to keep it aligned with the options available for the top_nav customization (disabling default entries, add custom entries)